azure/templates/post-deploy.yml

```diff
@@ -78,6 +78,34 @@ steps:
     displayName: Update endpoints KVM
     workingDirectory: "$(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/terraform"
 
+  - bash: |
+      set -e
+      if ! [[ $APIGEE_ENVIRONMENT =~ .*-*sandbox ]]; then
+          domain_name=$(make -s output name=service_domain_zone)
+          domain_name=`sed -e 's/^"//' -e 's/"$//' <<< $domain_name`
+          endpoint="https://${domain_name}/_status"
+          echo ${endpoint}
+
+          #Setting a counter to check if ECS Task is deployed + added sleep for 2 mins
+          counter=0
+          while [ $counter -lt 7 ]
```

Comment:
You could probably streamline this sleep loop by using Azure Pipeline's `retryCountOnTaskFailure` option, docs [here](https://learn.microsoft.com/en-us/azure/devops/release-notes/2021/sprint-195-update#automatic-retries-for-a-task). AFAIK it retries immediately so you'd probably still need to sleep on an unsuccessful request.